react/components/Statement/Atoms/VerbAtom.tsx

```diff
@@ -44,6 +44,8 @@ const VerbAtom: React.FC<Props> = ({
         <span className="pt3">{value.label}</span>
       )}
     </div>
+  ) : (
+    <span className="pt3" />
```

Comment:
Why not null or Fragment?

Reply:
Without the span to apply this margin it gets little bit weird:

![Captura de tela de 2020-05-14 11-13-50](https://user-images.githubusercontent.com/4912690/81945109-08b66d00-95d4-11ea-8585-d598d1e877ee.png)

I could fix that margin in the specific app I'm using it but probably would require to fix it in other apps that are using it too

Reply:
Why not solve this in `Statement` component?

Reply:
Because the value I need to display is calculated in this component and theres a lot of logic in the statement already

Reply:
The amount of logic in the `Statement` component is, in my opinion, irrelevant for this particular case. About the value calculated I agree that it looks like the logic should be inside but it doesn't. The `verbs.length > 0 && (<Verb />)` condition should solve the problem

Reply:
Ok, I did like that but theres the problem with the margin. I'll fix it in admin collections but I don't know what impact this will have in other apps

Reply:
I think that to solve this margin problem you could extract the element inside the `VerbAtom` to the `Statement` component. This way you could let the container with this margin and conditionally show the `VerbAtom`. What do you think?